### ISSUE TITLE
Pandas

### ISSUE THREAD

**Issue by wavexx** [firecat53/tabview#43]
I... would love too see tabview integration with pandas.
On both sides:

  $ tabview file

on the command line, bin/tabview.py should perform a try: import of pandas, and use read_* methods to read any data type currenly supported by pandas. We can wrap this using a generic load() method that tries pandas first, and then csv if nothing else is available. Unfortunately I couldn't find a pandas function to load a file without having to specify a file type. But we could maybe use file.magic() and then use the appropriate read_table/read_csv/read_excel/... functions.

On the other side, I would love to just do the following:

  import pandas as pd
  data = pd.read_table("file")
  data.view()

Anything against proposing this to the pandas developers?

**Comment by wavexx** [comment]
I'm too trigger-happy: https://github.com/pydata/pandas/issues/9179#issuecomment-68542977

**Comment by wavexx** [comment]
I did a first try on trying to integrate the tabview module into pandas.DataFrame.
So far my findings:

- sys.exit() on quit is not appropriate when using tabview as a module
- something funky is going on when using view() simply using ipython: the size of the terminal as returned by curses is wrong in my tests, and limited to what looks like ~25 lines.
- Aaahh... python 2.7. Should I say more?

**Comment by firecat53** [comment]
In my opinion, the integration should happen on the pandas side, or with a small helper script to convert file/data into the tabular list-of-lists needed by tabview.

I'll keep this open, as if there are small things I can do to improve integration with other projects, I'll try to accommodate...again keeping the ultimate goal of simplicity in mind. 

Yeah...python 2.7. <sigh> If I remember right, I went for python 3 initially because of improved Unicode handling. I've got a couple of other projects that are dual 2/3 compatible. I can see what would need to be fixed here to accomplish that.
